src/FakeItEasy.Analyzer.VisualBasic/FakeItEasy.Analyzer.VisualBasic.csproj

```diff
@@ -1,32 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <RootNamespace>FakeItEasy.Analyzer</RootNamespace>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <FileAlignment>512</FileAlignment>

```

Comment:
@thomaslevesque, I took your previous suggestion and dropped AppDesignFolder and FileAlignement when I consolidated. If it offends, I can restore and remove in a different PR.

Reply:
👍